portfolio/src/main/webapp/index.html

```diff
@@ -15,6 +15,7 @@
         <a href="#Contact" class="button" id="contact">Contact</button>
```

Comment:
Make sure to use `</a>` as the closing tag, since the opening tag is a button (same thing for previous line)